docs/troubleshooting-guide.md

```diff
@@ -70,7 +71,25 @@ To mitigate, please resize your VM in the resource group to a machine with at le
 
 ## Setup
 ### Setup Cromwell on Azure for multiple users in the same Azure subscription
-This section is COMING SOON.
+Cromwell on Azure is designed to be flexible for single and multiple user scenarios. Here we have envisioned 4 general scenarios and demonstrated how they relate to your Azure account, Azure Batch service, Subscription ID, and Resource Groups, each depicted below.
+
+![Multiple Users FAQ](/docs/screenshots/multiple-users-configs.png)
+
+1) **The Individual User**: This is the current standard deployment configuration for Cromwell on Azure. No extra steps beyond the Quick Start guide are necessary.
```

Comment:
Rename "Quick Start guide" to "deployment guide" and add link to https://github.com/microsoft/CromwellOnAzure#deploy-your-instance-of-cromwell-on-azure